go.mod

```
@@ -298,6 +299,9 @@ replace (
 	github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.16.0
 
+	// Avoid  CVE-2023-46402
+	github.com/whilp/git-urls v1.0.0 => github.com/chainguard-dev/git-urls v1.0.2
```

```suggestion
	github.com/whilp/git-urls => github.com/chainguard-dev/git-urls v1.0.2
```

Ah. That oughta do it.

---

Don't commit the suggestion here though, do it locally and then run go mod tidy.